### PR TITLE
created new column and deleted language_id

### PR DESCRIPTION
**Pull request by Zingerd**
created a new column and deleted language_id and also refactoring some methods

### CODE REVIEW

**Review by Max877866 — APPROVED** [review]

**Review by Bohdan-Melnyk — APPROVED** [review]

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 1]
```diff
@@ -446,17 +440,17 @@ void addTariffServiceExceptionTest() {
...
-        User user = ModelUtils.getUser();
-        AddServiceDto dto = ModelUtils.addServiceDto();
-
-        when(userRepository.findByUuid("123233")).thenReturn(user);
-        when(locationRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getLocation()));
-        when(locationRepository.findById(1L)).thenReturn(Optional.ofNullable(ModelUtils.getLocation()));
-
-        assertThrows(LanguageNotFoundException.class, () -> superAdminService.addTariffService(dto, "123233"));
-    }
+//    @Test
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: Remove comment's

**Review by VadymMakitra — CHANGES_REQUESTED** [review]

File: service/src/test/java/greencity/ModelUtils.java [hunk 2]
```diff
@@ -1877,7 +1878,8 @@ private static NotificationDto createNotificationDto() {
     public static List<TariffTranslationDto> getTariffTranslationDto() {
         return List.of(TariffTranslationDto.builder()
             .description("Test")
-            .languageId(1L)
+//            .languageId(1L)
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/ModelUtils.java]: remove comment's

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 3]
```diff
@@ -73,14 +73,13 @@ void addTariffServiceTest() {
 
         when(userRepository.findByUuid("123233")).thenReturn(user);
         when(locationRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getLocation()));
-        when(languageRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getLanguage()));
+//        when(languageRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getLanguage()));
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: !

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 4]
```diff
@@ -308,7 +307,7 @@ void includeBag() {
         BagTranslation bagTranslationTest = BagTranslation.builder()
             .id(2L)
             .bag(Bag.builder().id(2).capacity(120).price(350).build())
-            .language(Language.builder().id(1L).code("en").build())
+//            .language(Language.builder().id(1L).code("en").build())
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: !

**Review by VadymMakitra — CHANGES_REQUESTED** [review]

**Comment by VadymMakitra** [comment]
Please resolve comments!

**Review by thesvider — APPROVED** [review]

**Review by thesvider — CHANGES_REQUESTED** [review]
Resolve code smells

**Review by thesvider — APPROVED** [review]

**Review by thesvider — APPROVED** [review]

**Review by VadymMakitra — APPROVED** [review]

**Pull request merged**